docs/get_started.md

```diff
@@ -139,6 +139,8 @@ you can install it before installing MMCV.
 c. Some dependencies are optional. Simply running `pip install -v -e .` will
  only install the minimum runtime requirements. To use optional dependencies like `albumentations` and `imagecorruptions` either install them manually with `pip install -r requirements/optional.txt` or specify desired extras when calling `pip` (e.g. `pip install -v -e .[optional]`). Valid keys for the extras field are: `all`, `tests`, `build`, and `optional`.
 
+d. If you would like to install `albumentations`, we suggest using `pip install albumentations>=0.3.2 --no-binary imgaug,albumentations`, which will not install `opencv-python-headless`. If you have already installed `opencv-python-headless` instead of `opencv-python`, you can simple use `pip install albumentations>=0.3.2`. More details can follow [docs](https://albumentations.ai/docs/getting_started/installation/#note-on-opencv-dependencies).
```

Comment:
More details can follow [docs] -> Please refer to official documentation for more details.

Reply:
Suggest to add the installation command after line 128